spec/requests/movies_spec.rb

```diff
@@ -0,0 +1,337 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe '/movies', type: :request do
+  let(:user) { create :user }
+  let(:manager) { create :user, email: 'manager@gmail.com', role: :manager }
+  describe 'GET /movies' do
+    subject(:request) { get movies_url }
+
+    before { request }
+
+    it 'returns successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'renders proper template' do
+      expect(response).to render_template('movies/index')
+    end
+  end
+
+  describe 'GET /movies/movie_id' do
+    let(:movie) { create :movie }
+    it 'returns successful response' do
+      get("/movies/#{movie.id}")
+      expect(response.status).to eq(200)
+    end
+
+    it 'renders proper template' do
+      get("/movies/#{movie.id}")
+      expect(response).to render_template('movies/show')
+    end
```

Comment:
If you implement the change above you can then do
```suggestion
    let(:movie) { create :movie }
    subject(:request) { get("/movies/#{movie.id}") }
    it 'returns successful response' do
      expect(response.status).to eq(200)
    end

    it 'renders proper template' do
      expect(response).to render_template('movies/show')
    end
```
And similarly below